src/test/java/Homework16.java

```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public static void registrationNavigation() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        String url1 = "https://bbb.testpro.io/registration.php";
```

Comment:
This variable should be renamed to something more descriptive, like

`String urlRegistration = "https://bbb.testpro.io/registration.php";`

![image](https://user-images.githubusercontent.com/104761482/220852526-a1c4772b-b0e7-4130-b07c-2e6edfe7e194.png)